packages/foreman/rubygem-gettext/rubygem-gettext.spec

```diff
@@ -85,8 +85,6 @@ find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 %{gem_instdir}/bin
 %{gem_libdir}
 %{gem_instdir}/locale
-%{gem_instdir}/po
```

Comment:
Will this pass or do you need to use `%exclude`?

Reply:
probably need to exclude